docs/modules/ROOT/pages/advanced-how-tos/managing-security-fix.adoc

```diff
@@ -1 +1,100 @@
+* **Prepare the fix:** Preparing the fix with source code in your private git mirror and builds in your private workspace.
+* **Release the fix:** Releasing the build from your private workspace and "upstreaming" your patch from your private git mirror back to your team's normal development repo.
+
+== Cloning applications to a private workspace
+Cloning an application involves replicating the application's code and configuration into a dedicated private workspace, ensuring that sensitive information remains isolated and protected.
+
+.**Prerequisites**
+
+* You have the Maintainer role or greater in the original workspace where the embargoed issue was identified.
+// Maintainer role was xref:getting-started/roles_permissions/ Currently no such file in konflux-ci/docs. Add a link if the doc is published.
+* You have link:https://konflux-ci.dev/docs/getting-started/cli/[enabled CLI access for {ProductName}].
+
+.**Procedure**
+
+. Create a link:https://docs.github.com/en/repositories/creating-and-managing-repositories/duplicating-a-repository[Private Mirror] of the corresponding repository for any components that will receive fixes related to embargoed information. This step is crucial to safeguard sensitive data and prevent unauthorized access.
+
++
+NOTE: Creating a private fork of the public link:https://docs.github.com/en/repositories/creating-and-managing-repositories/duplicating-a-repository[GitHub repository] is not possible.
+
+. link:https://konflux-ci.dev/docs/getting-started/#for-developers[Request a new workspace] from your platform engineer. This workspace will serve as the private environment for cloning the affected application. Avoid adding information about the vulnerability to a workspace name.
+
++
+[IMPORTANT]
+====
+* If you already have a private workspace that you used for another embargoed issue, you can re-use it. Always double-check the link:https://console.redhat.com/preview/application-pipeline/access[User Access] permissions.
+
+* You can reuse a personal workspace, but double-check the User Access permissions.
+====
+
+. Log in to the original workspace where the application was previously being developed using the `oc` CLI tool. (See xref:getting-started/cli/[Getting started with CLI].) Append `/workspaces/<original-tenant-workspace>` to the AppStudio API URL used in the `oc login` command to target a particular workspace other than your default user workspace.
+// Add a link to the KUBECONFIG workflow when it's documented. Either approach is legit, up to a user to choose.
+. To export records the application from the original workspace with details changed to refer to your new private workspace, run the following command:
+
++
+[source,bash]
+----
+./rhtap export application <name_of_your_application>  --from <original-tenant-workspace> --to <private-workspace> --as-prebuilt-images --skip <impacted component git url>
```

Comment:
`rhtap` CLI is used here. I assume that this would still work ... but the naming is unfortunate as there is not currently a `konflux` CLI.